lib/app-components/addon/components/error-page/template.hbs

```diff
@@ -1,34 +1,34 @@
-{{title (t (concat translateKey '.heading'))}}
+{{title (t (concat this.translationKey '.heading'))}}
 
-<div class="container">
-    <div class="row">
-        <div class="col-xs-12">
-            <h1>{{t (concat translateKey '.heading')}}</h1>
+<div class='container'>
+    <div class='row'>
+        <div class='col-xs-12'>
+            <h1>{{t (concat this.translationKey '.heading')}}</h1>
```

Comment:
Because:
https://github.com/CenterForOpenScience/ember-osf-web/blob/c2c9afbd23c5290fa8b038c933da07146f637ad3/lib/app-components/addon/components/error-page/component.ts#L18
But actually, it seems to be invoked with `translateKey`:
https://github.com/CenterForOpenScience/ember-osf-web/blob/c2c9afbd23c5290fa8b038c933da07146f637ad3/lib/collections/addon/page-not-found/template.hbs#L1-L5
https://github.com/CenterForOpenScience/ember-osf-web/blob/c2c9afbd23c5290fa8b038c933da07146f637ad3/lib/collections/addon/forbidden/template.hbs#L1-L5
so the component doesn't match the template. We *really* need template type-checking!